packages/eas-cli/src/commands/build/cancel.ts

```diff
@@ -0,0 +1,129 @@
+import { Command } from '@oclif/command';
+import chalk from 'chalk';
+import gql from 'graphql-tag';
+import ora from 'ora';
+
+import { platformEmojis } from '../../build/constants';
+import { Platform } from '../../build/types';
+import { graphqlClient, withErrorHandlingAsync } from '../../graphql/client';
+import { AppPlatform, Build, BuildStatus } from '../../graphql/generated';
+import { BuildQuery } from '../../graphql/queries/BuildQuery';
+import log from '../../log';
+import {
+  findProjectRootAsync,
+  getProjectFullNameAsync,
+  getProjectIdAsync,
+} from '../../project/projectUtils';
+import { confirmAsync, selectAsync } from '../../prompts';
+
+async function cancelBuildAsync(buildId: string): Promise<Pick<Build, 'id' | 'status'>> {
+  const data = await withErrorHandlingAsync(
+    graphqlClient
+      .mutation<{ build: { cancel: { id: string; status: BuildStatus } } }>(
+        gql`
+          mutation CancelBuildMutation($buildId: ID!) {
+            build(buildId: $buildId) {
+              cancel {
+                id
+                status
+              }
+            }
+          }
+        `,
+        { buildId }
+      )
+      .toPromise()
+  );
+  return data.build!.cancel;
+}
+
+const appPlatformMap = {
```

Comment:
Move this to `build/constants.ts`

Reply:
I prefer to keep it here as long we have duplicate types for apiv2 and graphql api